sources/github.go

```diff
@@ -105,10 +105,12 @@ func (g *Github) alreadyDispatched(eventID string) bool {
 	return false
 }
 
-func (g *Github) fetchRepo(repo string) {
-	parts := strings.Split(repo, "/")
+func (g *GitHub) pullAndValidateEvents(parts []string) {
+	g.mu.Lock()
```

Comment:
Why the g.mu.Lock() and Unlock() are duplicated?

Reply:
thanks, looks like a typo